app/controllers/publishers/vacancies/job_applications/notes_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Publishers::Vacancies::JobApplications::NotesController < Publishers::Vacancies::JobApplications::BaseController
+  def index
+    @notes_form = Publishers::JobApplication::NotesForm.new
+  end
+
+  def create
+    @notes_form = Publishers::JobApplication::NotesForm.new(notes_form_params)
+
+    if @notes_form.valid?
+      Note.create(notes_form_params.merge(job_application: job_application, publisher: current_publisher))
+      flash[:success] = "A note has been added"
```

Comment:
This should be in the translations file

Reply:
Ah yeah forgot to do that. Cheers mate :)